### DISCUSSION TITLE
Axios (serverless dependency) high priority CSRF vulnerability, upgrade to 1.6.0

### DISCUSSION THREAD

**Discussion by jnovek** [serverless/serverless#12230] [General]
The `axios` package which is used by `serverless/platform-client` had a high-priority CSRF vulnerability flagged by snyk this morning.  Here's the snyk report:

https://security.snyk.io/vuln/SNYK-JS-AXIOS-6032459

Looks like the CVE is still a WIP.

Is this repo the appropriate place to report this issue or is there a separate issue board for `platform-client`?

**Comment by filbertasurion** [comment]
Any updates on this?

**Comment by jnovek** [comment]
Yup!  Here's the ticket --

https://github.com/serverless/serverless/issues/12232

Resolved in 3.37.
